### DISCUSSION TITLE
HTML markup for Lively Wallpaper

### DISCUSSION THREAD

**Discussion by rovdyX** [rocksdanister/lively#2094] [Q&A]
Hello, I am trying to make a video as a live wallpaper through HTML markup and I have a question why Lively Wallpaper does not see the block  video : <video class="video_media" src="res/video/movi.mp4"
autoplay loop muted></video>

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Web-Guide-VI-:-Webpage-Video-Support
